pkg/apis/networking/v1alpha1/ingress_defaults.go

```diff
@@ -35,7 +35,9 @@ func (is *IngressSpec) SetDefaults(ctx context.Context) {
 	for i := range is.Rules {
 		is.Rules[i].SetDefaults(ctx)
 	}
-
+	if is.HTTPOption == "" {
+		is.HTTPOption = HTTPOptionEnabled
+	}
```

Comment:
This may cause downgrade test failures 🤔

Reply:
I suspect this change is making "upgrade" testing fail here: https://github.com/knative/serving/pull/10386

Reply:
Yeah see net-istio downstream failures too.